cmd/isolateDAG.go

```diff
@@ -203,6 +203,15 @@ func isolateGraph(graph *fs.Graph) {
 					fmt.Printf("❌ Unable to touch %s `%s`: %s\n", upstream.Type, upstream.Name, err)
 					return err
 				}
+				// Symlink the schema if it exists
+				schemaFile := strings.TrimSuffix(upstream.Path, filepath.Ext(upstream.Path)) + ".yml"
+				if _, err := os.Stat(schemaFile); upstream.Schema != nil && err == nil {
+					if err := symLink(schemaFile); err != nil {
+						pb.Stop()
+						fmt.Printf("❌ Unable to isolate schema for upstream model %s `%s`: %s\n", upstream.Type, upstream.Name, err)
+						return err
+					}
+				}
```

Comment:
Could we make this a function in here as this code is now copied twice?

something like

```go
symLinkSchema := func(file *fs.File) error {
...
}
```